src/display/api.js

```diff
@@ -2944,6 +2953,12 @@ class WorkerTransport {
     return this.messageHandler.sendWithPromise("GetAttachments", null);
   }
 
+  getEditableAnnotations(pageIndexToSkip) {
```

Comment:
Here and elsewhere in this patch, what is the purpose of the `pageIndexToSkip` parameter? I don't see it being used anywhere AFAICT, so is it necessary to introduce this already?

Reply:
The function itself isn't used yet but will be in a couple of days, for example to get all the comments (popups) which are in a pdf. I added it a this separate commit in order to not have a too big patch.
About `pageIndexToSkip`, when switching to editing mode with page 1 visible, it'll avoid to get the annotations for this page since we already have them from the annotation layer.
My goal is to almost be able to get all the invisible (aka non-rendered) annotations.